en/docs/security-announcements/cve-justifications/2025/CVE-2025-46392.md

```diff
@@ -0,0 +1,30 @@
+---
+title: "CVE-2025-46392"
+category: security-announcements
+date: July 31 2025
+---
+
+# CVE-2025-46392
+
+<p class="doc-info">WSO2 Products impacted: no</p>
+<p class="doc-info">Customers actions required: no</p>
+---
+
+### REPORTED VULNERABILITY
+
+A Denial of Service (DoS) vulnerability in Apache Commons Configuration 1.x which can be exploited by supplying a specially crafted configuration file containing malicious or recursive variable interpolation, leading to uncontrolled resource consumption.. [^1]
+
+### REPORTED PRODUCTS
+
+- WSO2 Identity Server : 7.0.0, 7.1.0
+
+### WSO2 JUSTIFICATION
+
+Upon evaluating the impact of upgrading Apache Commons Configuration from version 1.x to 2.x, we identified significant architectural changes that introduce compatibility issues. Version 1.x uses the org.apache.commons.configuration.* package hierarchy, whereas version 2.x has been restructured under org.apache.commons.configuration2.*, with major modifications. Additionally, to make commons-configuration2 function as an OSGi service within the product, several new dependencies must be introduced, increasing the integration complexity. Due to these concerns, we are publishing this CVE justification with a detailed analysis of the CVE, outlining how associated risks are mitigated in WSO2 products.
+
+The reported vulnerability involves uncontrolled resource consumption via malicious or recursive variable interpolation in configuration files. In WSO2 Identity Server, the affected component is used exclusively to load the log4j2.properties file. This file is not user-controllable under normal deployment scenarios.
```

Comment:
```suggestion
The reported vulnerability involves uncontrolled resource consumption via malicious or recursive variable interpolation in configuration files. In WSO2 Identity Server, the affected component is used exclusively to load the `log4j2.properties` file. This file is not user-controllable under normal deployment scenarios.
```

> normal deployment scenarios.
Can there be any other scenarios? If not, shall we remove "normal" word from here?